content/en/docs/contribute/localization.md

```diff
@@ -14,34 +14,34 @@ card:
 <!-- overview -->
 
 This page shows you how to
-[localize](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/) the docs for a
-different language.
+[localize](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/)
+the docs for a different language.
 
 
 <!-- body -->
 
 ## Contribute to an existing localization
 
-You can help add or improve content to an existing localization. In [Kubernetes
-Slack](https://slack.k8s.io/) you'll find a channel for each localization. There is also a general
-[SIG Docs Localizations Slack channel](https://kubernetes.slack.com/messages/sig-docs-localizations)
-where you can say hello.
+You can help add or improve the content of an existing localization. In
+[Kubernetes Slack](https://slack.k8s.io/), you can find a channel for each
+localization. There is also a general [SIG Docs Localizations Slack
+channel](https://kubernetes.slack.com/messages/sig-docs-localizations) where you
+can say hello.
 
-{{< note >}}
-If you want to work on a localization that already exists, check
-this page in that localization (if it exists), rather than the
-English original. You might see extra details there.
-{{< /note >}}
+{{< note >}} For extra details on how to contribute to a specific localization,
+look for a localized version of this page. {{< /note >}}
 
 ### Find your two-letter language code
 
-First, consult the [ISO 639-1 standard](https://www.loc.gov/standards/iso639-2/php/code_list.php)
-to find your localization's two-letter language code. For example, the two-letter code for Korean
-is `ko`.
+First, consult the [ISO 639-1
+standard](https://www.loc.gov/standards/iso639-2/php/code_list.php) to find your
+localization's two-letter language code. For example, the two-letter code for
+Korean is `ko`.
 
 ### Fork and clone the repo
 
-First, [create your own fork](/docs/contribute/new-content/open-a-pr/#fork-the-repo) of the
+First, [create your own
+fork](/docs/contribute/new-content/open-a-pr/#fork-the-repo) of the
```

Comment:
Do not break a hyperlink if possible; we have a script to validate these links and that script is not smart enough.